llvm_util/llvm2alive.cpp

```diff
@@ -579,12 +566,12 @@ class llvm2alive_ : public llvm::InstVisitor<llvm2alive_, unique_ptr<Instr>> {
         return error(i);
     }
 
-    unordered_set<llvm::Value*> visited;
     // FIXME: size bits shouldn't be a constant
     auto size = make_intconst(DL().getTypeAllocSize(i.getAllocatedType()), 64);
-    RETURN_IDENTIFIER(make_unique<Alloc>(*ty, value_name(i), *size, mul,
-                        pref_alignment(i, i.getAllocatedType()),
-                        hasLifetimeStart(i, visited)));
+    auto alloc = make_unique<Alloc>(*ty, value_name(i), *size, mul,
+                      pref_alignment(i, i.getAllocatedType()));
+    allocs.emplace(&i, make_pair(alloc.get(), /*has lifetime.start?*/ false));
```

Comment:
This flag will be turned on by `getLifetimeKind` if it finds lifetime.start with a pointer to this alloca.